README.md

```diff
@@ -3,15 +3,16 @@
 This Web3.js plugin adds support for the following wallet-related RPC methods:
 
 - [wallet_addEthereumChain (EIP-3085)](https://eips.ethereum.org/EIPS/eip-3085)
-- [wallet_updateEthereumChain (EIP-2015)](https://eips.ethereum.org/EIPS/eip-2015)
 - [wallet_switchEthereumChain (EIP-3326)](https://eips.ethereum.org/EIPS/eip-3326)
-- [wallet_getOwnedAssets (EIP-2256)](https://eips.ethereum.org/EIPS/eip-2256)
 - [wallet_watchAsset (EIP-747)](https://eips.ethereum.org/EIPS/eip-747)
-
-Not implemented yet:
-
 - [wallet_requestPermissions (EIP-2255)](https://eips.ethereum.org/EIPS/eip-2255)
 - [wallet_getPermissions (EIP-2255)](https://eips.ethereum.org/EIPS/eip-2255)
+- [wallet_revokePermissions](https://docs.metamask.io/wallet/reference/json-rpc-methods/wallet_revokepermissions/)
+
+Experimental:
```

Comment:
It would be nice to explain why these methods are considered experimental.

Reply:
done per function in the examples below, here is the link to the full EIP

Reply:
done